blueprints/ember-frost-popover/index.js

```diff
@@ -3,7 +3,7 @@ const blueprintHelper = require('ember-frost-core/blueprint-helper')
 module.exports = {
   afterInstall: function (options) {
     const addonsToAdd = [
-      {name: 'ember-frost-core', target: '3.0.0'}
+      {name: 'ember-frost-core', target: '^3.0.0'}
```

Comment:
This is not related to the context of this PR.  Do not mix efforts.  Remove this change.